lib/runPrettier.js

```diff
@@ -0,0 +1,61 @@
+const { spawn } = require('child_process');
+const path = require('path');
+
+const cwd = process.cwd();
+
+const filePattern = 'src/**/*.js';
+
+function runPrettier({ write, listDifferent, config }) {
+  const prettierConfig = config || {};
+
+  const prettierPath = path.join(
+    path.resolve('node_modules/prettier'),
```

Comment:
As discussed, require.resolve will resolve to the filename directed to by the packages package.json `main` attribute, and I'm trying to find the bin file.
I'd like to leave it as is, but perhaps some kind of reference to %PATH% would be more appropriate to find bin files in the future, not sure and needs discussion.